common/static/js/fixtures/lang-edx-fixture.html

```diff
@@ -1,5 +1,5 @@
 <form action="/i18n/setlang/" method="post" class="settings-language-form" id="language-settings-form">
-    <input title="preference api" type="hidden" id="preference-api-url" value="/api/user/v1/preferences/test1/">
+    <input title="preference api" class="url-endpoint" type="hidden" id="preference-api-url" value="/api/user/v1/preferences/test1/" data-user-is-authenticated="true">
```

Comment:
We are using this in `common/static/js/spec/lang_edx_spec.js` , These tests were written to support WL-243 (Language selector in LMS). and  `common/static/js/src/lang_edx.js` is using the `url-endpoint` class